python3.6-slim/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM python:3.6-slim
+
+# set the locale
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+# Create working directory
+ENV WORKING_DIR=/opt/invenio
+ENV INVENIO_INSTANCE_PATH=${WORKING_DIR}/var/instance
+RUN mkdir -p ${INVENIO_INSTANCE_PATH}
+
+# Create files mountpoints
+RUN mkdir ${INVENIO_INSTANCE_PATH}/data
+RUN mkdir ${INVENIO_INSTANCE_PATH}/archive
+RUN mkdir ${INVENIO_INSTANCE_PATH}/static
+
+# copy everything inside /src
+RUN mkdir -p ${WORKING_DIR}/src
+WORKDIR ${WORKING_DIR}/src
+
+# Set `npm` global under Invenio instance path
+RUN mkdir ${INVENIO_INSTANCE_PATH}/.npm-global
+ENV NPM_CONFIG_PREFIX=$INVENIO_INSTANCE_PATH/.npm-global
+
+# required debian packages
+RUN set -ex \
+  && buildDeps=' \
+    curl \
+    gcc \
+    libc6-dev \
+    libffi-dev \
+    libxml2-dev \
+    libxslt1-dev \
+  ' \
+  && apt-get update \
+  && apt-get install --no-install-recommends -y \
+    git \
```

Comment:
We are missing `python-cairosvg` which is required by Invenio-Formatter to generate badges (it is [part of the Invenio base bundle](https://github.com/inveniosoftware/invenio/blob/9f5f7f448dea7669d83388a355f7db7c9d35d074/setup.py#L37)).

Reply:
I add it in "persistent" dependencies.